src/main/kotlin/com/quipalup/katydid/logentry/domain/LogEntryRepository.kt

```diff
@@ -13,9 +12,5 @@ interface LogEntryRepository {
 
 // parallel change
 interface LogEntryRepositoryPC {
-    fun findById(id: Id): Either<FindLogEntryError, LogEntry_>
-    fun deleteById(id: Id): Either<DeleteLogEntryError, Unit>
-    fun save(logEntry: LogEntry_): Either<SaveLogEntryError, Id>
-    fun existsById(id: Id): Boolean
-    fun saveAll(logEntries: List<LogEntry_>): Either<CreateLogEntriesError, List<Id>>
+    fun searchAllByChildId(childId: ChildId): List<LogEntry_>
```

Comment:
What about using `Id` class for all id entities in backend?

Reply:
I rather have meaningful value objects :)